showcase/src/components/ui/thread-history.tsx

```diff
@@ -240,6 +250,7 @@ const ThreadHistoryNewButton = React.forwardRef<
 
       try {
         await startNewThread();
+        await refetch();
```

Comment:
Hmm I wish there was a clean way to merely invalidate this rather than having to call refetch, but I know we do that elsewhere.